ci/vcpkg/ports.patch

```diff
@@ -49,18 +49,5 @@ index 0000000000..a57ce0c22f
 +   return v & ~(mask << (8 * n));
 +-#endif
 + }
-+ 
++
 + static inline bool LeftShiftOverflows(uint8_t value, uint32_t shift) {
-diff --git a/scripts/cmake/vcpkg_find_acquire_program.cmake b/scripts/cmake/vcpkg_find_acquire_program.cmake
-index 4611af6..d11936f 100644
---- a/scripts/cmake/vcpkg_find_acquire_program.cmake
-+++ b/scripts/cmake/vcpkg_find_acquire_program.cmake
-@@ -239,7 +239,7 @@ function(vcpkg_find_acquire_program program)
-             set(paths_to_search "${DOWNLOADS}/tools/python/${tool_subdirectory}")
-             vcpkg_list(SET post_install_command "${CMAKE_COMMAND}" -E rm python310._pth)
-         else()
--            set(program_name python3)
-+            set(program_name python)
-             set(brew_package_name "python")
-             set(apt_package_name "python3")
-         endif()
```

Comment:
The above patch no longer cleanly applies due to https://github.com/microsoft/vcpkg/pull/30780.

Having removed it, the following local builds succeed.

```bash
$ ARCH=amd64 PYTHON=3.10 archery docker run python-wheel-manylinux-2014
$ ARCH=amd64 PYTHON=3.10 archery docker run python-wheel-manylinux-2-28
```

However, it's probably worth running the entire suite to check if everything is OK

Reply:
I'll submit crossbow builds once the initial CI checks complete

Reply:
Oh, sorry. I've done it!

Reply:
> Oh, sorry. I've done it!

Thank you!